src/utils/exchange.js

```diff
@@ -105,7 +105,7 @@ export const parseOffer = (
 };
 
 export const isWethConvertedTx = (fromAssetSymbol: string, contractAddress: string): boolean => {
-  return fromAssetSymbol === ETH && contractAddress === WETH[chainId].address;
+  return fromAssetSymbol === ETH && contractAddress === WETH[getChainId()].address;
 };
```

Comment:
It seems like we have general purpose (exchange) `isWethConvertedTx` function, relying on uniswap specific `getChainId` function. Either we want to have both function under `uniswap` or both functions under `exchanges`. Wdyt?